hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/recon/ReconConfigKeys.java

```diff
@@ -40,6 +40,12 @@ private ReconConfigKeys() {
       "ozone.recon.http-address";
   public static final String OZONE_RECON_HTTPS_ADDRESS_KEY =
       "ozone.recon.https-address";
+  public static final String OZONE_RECON_ADDRESS_DEFAULT =
+      "recon:9891";
```

Comment:
Recon RPC address default host should not be `recon`. This default value isn't just for the testing environment. `0.0.0.0` should allow Recon server to start up normally and listen on all interfaces.

But we should expect admins to set this address properly for **clients**.

```suggestion
      "0.0.0.0:9891";
```